tests/utils/get-learn-page-filters.test.js

```diff
@@ -0,0 +1,85 @@
+import { getLearnPageFilters } from '../../src/utils/setup/on-create-page';
+
+it('should correctly create filters for the learn page based on article tags', () => {
+    const allArticles = [
+        {
+            query_fields: {
+                languages: ['node', 'react'],
+                products: ['atlas', 'stitch'],
+            },
+        },
+        {
+            query_fields: {
+                languages: ['python', 'react'],
+                products: ['server'],
+            },
+        },
+        {
+            query_fields: {
+                languages: ['python', 'react'],
+                products: ['server'],
+            },
+        },
+        {
+            query_fields: {
+                languages: [],
+                products: ['server', 'stitch'],
+            },
+        },
+    ];
+
+    const result = getLearnPageFilters(allArticles);
+    const languages = result.languages;
+    const products = result.products;
```

Comment:
nit suggestion - check for result.language/products right in the expect rather than assigning. It'll draw a more clear line from the work to the test. I'd also move the `result` assignment right above the `expect`s